intro-HPC/ch_account.tex

```diff
@@ -94,18 +94,6 @@ \subsection{How do SSH keys work?}
   You can download PuTTY from the official address:
   \url{http://www.chiark.greenend.org.uk/\~sgtatham/putty/}
 
-  The PuTTY package consists of several components:
-
-  \begin{enumerate}
-    \item  \strong{PuTTY}: the Telnet and SSH client itself
-    \item  \strong{PSCP}: an SCP client, i.e., command-line secure file copy
-    \item  \strong{PSFTP}: an SFTP client, i.e., general file transfer sessions much like FTP
-    \item  \strong{Plink}: a command-line interface to the PuTTY back ends
-    \item  \strong{Pageant}: an SSH authentication agent for PuTTY, PSCP and Plink
-    \item  \strong{PuTTYgen}: an RSA and DSA key generation utility
-    \item  \strong{pterm}: a standalone terminal emulator
-  \end{enumerate}
```

Comment:
Don't remove this as a whole, highlight the relevant parts: `PyTTYgen` (to generate key pair, ref to section) + `PuTTY` (to login in, ref to section).

Also mention that *installing* is not actually needed, you can just download the `.exe` and run it (that's useful if you don't have admin privileges).